src/containers/AccountConnection.jsx

```diff
@@ -420,7 +419,6 @@ class AccountConnection extends Component {
             deleting={deleting}
             disableSuccessTimeout={disableSuccessTimeout}
             displayAdvanced={displayAdvanced}
-            driveUrl={driveUrl}
```

Comment:
OK I understand now I think what standalone meant. I think those changes should be fixed up with the commit "Standalone KonnectorFolder"